mypy/lex.py

```diff
@@ -740,15 +747,18 @@ def lex_break(self) -> None:
             last_tok.string += self.pre_whitespace + s
             self.i += len(s)
             self.line += 1
+            self.column = 0
```

Comment:
why the assignment of 0? the value of `i`, or 1 character beyond the end of the string seems to make sense for describing a line break